static/library/highlight_component.ts

```diff
@@ -61,6 +66,10 @@ export class DatacommonsHighlightComponent extends LitElement {
   @property()
   place!: string;
 
+  // Optional: List of variable DCIDs to plot per capita
+  @property({ type: Array<string>, converter: convertArrayAttribute })
+  perCapita?: string[];
+
```

Comment:
Since highlight/gauge only take one variable, would we want to make this a boolean for highlight/gauge? Or keep the perCapita spec uniform across all tiles?

Reply:
I went back and forth on this and and ended up going with uniform across the board. It's a little awkward for the single variable case, but i think i like the uniform appraoch. wdyt?